ICWProgressives.csv

```diff
@@ -1360,8 +1367,8 @@ Dathomir,2,Zsinj_Empire,,,1
 Dathomir,3,Rebel,Generate_Force,1,
 Dathomir,4,Rebel,,,3
 Dathomir,5,Rebel,,,3
-Dathomir,6,Rebel,,,3
-Dathomir,7,Rebel,,,3
+Dathomir,6,Hapes_Consortium,Generate_Force,1,

```

Comment:
🤔

Reply:
> 🤔

https://github.com/ThrawnsRevenge/ICWBeta/issues/1789
https://github.com/ThrawnsRevenge/ICWBeta/issues/1790

Reply:
It's definitely pretty cool thing for hapan later era thing hehe